programs/voter-stake-registry/src/instructions/accelerate_vesting.rs

```diff
@@ -0,0 +1,47 @@
+use crate::error::*;
+use crate::state::*;
+use anchor_lang::prelude::*;
+
+#[derive(Accounts)]
+pub struct AccelerateVesting<'info> {
```

Comment:
Naming: This instruction doesn't just accelerate vesting, it finishes vesting. But actually it's broader than that: it also removes any type of non-vesting lockup. Maybe "UnlockDeposit" would be appropriate?

Reply:
I like that, renamed everything